src/umpire/strategy/DynamicPoolHeuristic.cpp

```diff
@@ -43,7 +43,7 @@ std::function<bool(const strategy::DynamicPool&)> heuristic_percent_releasable(
   float f = (float)((float)percentage / (float)100.0);
 
   return [=] (const strategy::DynamicPool& pool) {
-      const long threshold = f * pool.getActualSize();
+      const std::size_t threshold = static_cast<std::size_t>(f * pool.getActualSize());
```

Comment:
Can you a comment like "Calculate threshold in bytes from the percentage"? I had to read this twice to make sure the casting was correct.

Reply:
don